From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jelle Raaijmakers <jelle@gmta.nl>
Date: Sun, 11 Feb 2024 23:47:48 +0100
Subject: [PATCH] Build: Do not hide symbols by default

For some reason, the functions glu exports end up as LOCAL entries in
the shared library. Remove this default visibility to set them to
GLOBAL.
---
 src/meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/meson.build b/src/meson.build
index 57d3b4a6163828445872f73ca4a92f2d202ac768..9ee969636950760354fb8990cc0bb7ea789098b5 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -103,7 +103,6 @@ libglu = library(
     'libnurbs/nurbtess',
     inc_include,
   ],
-  gnu_symbol_visibility : 'hidden',
   dependencies : [dep_gl],
   version : '1.3.1',
   darwin_versions  : [ '5.0.0', '5.1.0' ],
